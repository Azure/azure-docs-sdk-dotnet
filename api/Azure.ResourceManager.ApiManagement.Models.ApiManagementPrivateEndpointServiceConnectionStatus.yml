### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus&text=ApiManagementPrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.#ctor(System.String)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.op_Equality(Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus,Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.op_Implicit(System.String)~Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.op_Inequality(Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus,Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.Equals(Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.ToString
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.Approved
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.Pending
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.Rejected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus
commentId: T:Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus
namespace: Azure.ResourceManager.ApiManagement.Models
name: ApiManagementPrivateEndpointServiceConnectionStatus
fullName: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus
nameWithType: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.ApiManagement.dll
packagesWithMoniker:
- value: Azure.ResourceManager.ApiManagement v1.0.1
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.ApiManagement v1.1.0-beta.1
  monikers:
  - azure-dotnet-preview
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct ApiManagementPrivateEndpointServiceConnectionStatus : IEquatable<Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus>'
- lang: fsharp
  values:
  - value: type ApiManagementPrivateEndpointServiceConnectionStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure ApiManagementPrivateEndpointServiceConnectionStatus

      Implements IEquatable(Of ApiManagementPrivateEndpointServiceConnectionStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
summary: The private endpoint connection status.
metadata:
  api_name:
  - Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus
  api_location:
  - Azure.ResourceManager.ApiManagement.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus
  - Azure::ResourceManager::ApiManagement::Models::ApiManagementPrivateEndpointServiceConnectionStatus
  - ApiManagementPrivateEndpointServiceConnectionStatus
  helpviewer_keywords:
  - Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus struct [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
