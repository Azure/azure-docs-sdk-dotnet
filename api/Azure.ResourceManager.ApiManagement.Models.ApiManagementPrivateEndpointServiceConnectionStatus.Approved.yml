### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus&text=ApiManagementPrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.Approved
  commentId: P:Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.Approved
  nameWithType: ApiManagementPrivateEndpointServiceConnectionStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As ApiManagementPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.Approved*
commentId: Overload:Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.Approved
namespace: Azure.ResourceManager.ApiManagement.Models
name: Approved
fullName: Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.Approved
nameWithType: ApiManagementPrivateEndpointServiceConnectionStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.ApiManagement.dll
packagesWithMoniker:
- value: Azure.ResourceManager.ApiManagement v1.0.1
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.ApiManagement v1.1.0-beta.1
  monikers:
  - azure-dotnet-preview
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.Approved
  - Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.get_Approved
  api_location:
  - Azure.ResourceManager.ApiManagement.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.ApiManagement.Models.ApiManagementPrivateEndpointServiceConnectionStatus.Approved
  - Azure::ResourceManager::ApiManagement::Models::ApiManagementPrivateEndpointServiceConnectionStatus::Approved
  - Approved
  - ApiManagementPrivateEndpointServiceConnectionStatus.Approved
  - ApiManagementPrivateEndpointServiceConnectionStatus::Approved
  helpviewer_keywords:
  - ApiManagementPrivateEndpointServiceConnectionStatus.Approved property [.NET]
  - Approved property [.NET], struct ApiManagementPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
